aws_lambda_builders/workflows/provided_make/workflow.py

```diff
@@ -0,0 +1,50 @@
+"""
+ProvidedMakeWorkflow
+"""
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from aws_lambda_builders.actions import CopySourceAction
+from aws_lambda_builders.path_resolver import PathResolver
+from .actions import ProvidedMakeAction
+from .utils import OSUtils
+from .make import SubProcessMake
+
+
+class ProvidedMakeWorkflow(BaseWorkflow):
+
+    """
+    A Lambda builder workflow for provided runtimes based on make.
+    """
+
+    NAME = "ProvidedMakeBuilder"
+
+    CAPABILITY = Capability(language="provided", dependency_manager=None, application_framework=None)
+
+    EXCLUDED_FILES = (".aws-sam", ".git")
+
+    def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=None, osutils=None, **kwargs):
+
+        super(ProvidedMakeWorkflow, self).__init__(
+            source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=runtime, **kwargs
+        )
+
+        self.os_utils = OSUtils()
+
+        # Find the logical id of the function to be built.
+        options = kwargs.get("options") or {}
+        build_logical_id = options.get("build_logical_id", None)
```

Comment:
What happens when this is not provided?

Should we just error out here if it is a requirement of the builder?

Reply:
yeah we would error out saying that the required make file target was not present.